recipes/samarium/all/patches/1.0.0-RandomGenerator.cpp-link.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/src/samarium/util/RandomGenerator.cpp b/src/samarium/util/RandomGenerator.cpp
```

Comment:
Why is this patch required? 🤔 I've never used this library so appologies if my question is silly🙊 

same for the other static patch, did you get them from somewhere?

Reply:
No, I messed up where I put the implementation of some functions 😥.
I get link errors without this fix.